src/main/java/com/google/sps/data/Keywords.java

```diff
@@ -136,5 +145,22 @@ private static AnalyzeEntitiesResponse analyzeEntity(String message) throws IOEx
       ex.printStackTrace();
       return null;
     }
-  }  
+  }
+  
+  /**
+   * @return a map of keys to their list of keywords of all sets of keywords that the
```

Comment:
I'm having a bit of trouble parsing "their list of keywords of all sets of keywords".  Can you clarify how the list of keywords associated with each key is chosen.

Reply:
The list of keywords associated to each key is gotten from the NLP API, which extracts salient terms from the user's input. The key is automatically generated upon creation of a new Entity object (which we do before setting its properties and putting the entity into the datastore for later retrieval), so we use that automatically-generated key instead. But, I do agree that the Javadoc is confusing, and I will make it more clear what it does.